go/oasis-net-runner/cmd/root.go

```diff
@@ -139,6 +139,10 @@ func runRoot(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("root: failed to instantiate fixture: %w", err)
 	}
 
+	// Set logging level and format for all spawned nodes.
+	net.Config().NodeLogLevel = viper.GetString(cfgLogLevel)
```

Comment:
Feels like this doesn't belong here. It should be in `GetFixture` or together with `var logFmt logging.Format`, not sure. 

Are we abusing `cfgLogFmt` and `cfgLogLevel` here and using them for two different purposes?

Reply:
No, I'm using the same log level and format here on purpose (that's why this code is also here and not in `fixtures.go`) -- the idea is to set just the log level and format for the net runner and it gets automatically inherited for all the nodes it spawns.  This is way more convenient than having a separate flag just for this or setting the log config for each node individually.